lms/static/sass/views/_verification.scss

```diff
@@ -1020,15 +1021,15 @@
 
     .help-inline {
       @extend %t-copy-sub1;
-      float: left;
+      @include float(left);
       width: flex-grid(6,12);
-      margin: ($baseline*0.75) flex-gutter() 0 0;
+      @include margin(($baseline*0.75), flex-gutter(), 0, 0);
     }
 
     .wizard-steps {
-      float: right;
-      width: flex-grid(6,12);
-      text-align: right;
+      // @include float(right);
```

Comment:
If they aren't needed anymore, you can just remove these rules instead of commenting them out.